src/tools/perf/perftest.c

```diff
@@ -332,6 +332,29 @@ static ucs_status_t setup_sock_rte_loopback(struct perftest_context *ctx)
     return UCS_OK;
 }
 
+void backup_local_params(const perftest_params_t *params,
+                         perftest_params_local_backup_t *backup)
```

Comment:
maybe rename to `perftest_param_backup_local/restore()` and reuse `perftest_params_t` instead of adding `perftest_params_local_backup_t`?

Reply:
I don't think that we need to backup entire `perftest_params_t`, reusing the structure may lead to wrong assumption that all field are valid after backup.

Reply:
since functions are now in header, shall we add `perftest_params_*()` prefix to all of them?